rune/libenclave/epm/epm.go

```diff
@@ -51,7 +53,9 @@ func GetCache(ID string) *v1alpha1.Enclave {
 	Type := "enclave-cache-pool"
 	cacheResp, err := c.GetCache(ctx, &v1alpha1.GetCacheRequest{Type: Type, ID: ID})
 	if err != nil {
-		logrus.Fatalf("could not get cache from enclave cache pool: %v", err)
+		syscall.Unlink(sockpath)
```

Comment:
The delete operation should only occur in recvFd(). Use golang `defer` syntax to delete the sock as long as recvFd is failed.

Reply:
ok, thanks